### ISSUE TITLE
GalacticFreeStore (AI) - AI file doesn't service properly due to misspelled variable

### ISSUE THREAD

**Issue by AlyMar1994** [AlyMar1994/UFoCP#130]
**Quick Information:**
_OS:_ Windows 11
_FoC version:_ v1.121.786192
_UFoCP version:_ INDEV
_List any other mods used:_ N/A
_Were you playing on Steam?_ YES
_Were you in multiplayer?_ NO

**Explanation:**
(From EvilBobTheBob, on Steam/Modder's Hub):

"_Sometimes Petroglyph has misspelled variables that are like that because they are hardcoded. In this case, the vanilla GalacticFreeStore has a serious error. The UnitServiceRate variable is misspelled as UnitServciceRate. This prevents units from naturally moving from planet to planet using this interval._"

In short: the UnitServiceRate is an unspecified rate, so On_Unit_Service(object) doesn't properly get serviced every 8(?) seconds like it should, and may lead to irregular unit moving.

**Possible Cause:**
See above.

**Screenshots/Video:**
![image](https://github.com/AlyMar1994/UFoCP/assets/20891893/ffb2e941-01cb-4b6a-924d-0d0844cce722)
